Code/Mantid/docs/source/algorithms/ApplyCalibration-v1.rst

```diff
@@ -15,8 +15,7 @@ updated as absolute positions and so this update can be repeated.
 The PositionTable must have columns *Detector ID* and *Detector
 Position*. The entries of the *Detector ID* column are integer referring
 to the Detector ID and the enties of the *Detector Position* are
-`V3Ds <http://www.mantidproject.org/V3D>`_ referring to the position of the detector whose ID is in
-same row.
+V3Ds referring to the position of the detector whose ID is in same row.
```

Comment:
I think this should point at `mantid.kernel.V3D` or just `V3D` directly. I'm not sure which.